cli/initiator/initiator.go

```diff
@@ -237,6 +320,19 @@ var StartDKG = &cobra.Command{
 		if err != nil {
 			logger.Warn("Failed writing keyshares file: ", zap.Error(err))
 		}
+		if privKeyPath == "" && generateInitiatorKey {
+			rsaKeyPath := fmt.Sprintf("%s/encrypted_private_key-%v.json", keysharesOutputPath, depositData.PubKey)
```

Comment:
that's a good way to use the same output path, maybe we should also combine the deposit and keyshares path args to just one arg `outputPath`

Reply:
Yeah, I was thinking about that as well. Will make only one output path.